governance.md

```diff
@@ -54,7 +54,7 @@ Qualification guidelines
 * Has reviewed 5 PRs of significant scope
 * Sponsored by 2 Approvers
 
-[Current Approvers](https://github.com/spinnaker/spinnaker/blob/master/approvers.md)
+[Current Approvers](https://github.com/spinnaker/community/blob/master/membership.yml)
```

Comment:
@dorbin It _seems_ reasonably clear that "oss-reviewers" and "oss-approvers" are the sections readers should find, but there's e.g. also a "oss-kayenta-approvers" section that could cause confusion.

Perhaps something like `[Current Approves](...) (see oss-approvers)` or so?

Reply:
Yes, like that. But "...(see the `oss-approvers` section)."

Also, prefer lower-case "approvers" and "reviewers".

Reply:
> Also, prefer lower-case "approvers" and "reviewers".

"Approvers" seems capitalized elsewhere in the doc too, and appears to be being regarded as a specific title?

Reply:
```suggestion
[Current Approvers](https://github.com/spinnaker/community/blob/master/membership.yml) (see the `oss-approvers` section)
```